src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -761,6 +764,25 @@ bool primitive_inst::has_inner_networks() const {
     return (_impl_params->inner_nets.size() > 0);
 }
 
+void primitive_inst::do_runtime_handle_empty_inputs() {
```

Comment:
It seems that we don't need this additional handle, how about just do on_execute() at the returning point for empty WI? on_execute is for reuse inputs for optimizable nodes and empty tensor is optimizable.

Reply:
Updated. Thanks!